src/app.js

```diff
@@ -1 +1,25 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const createFile = () => {
+  const date = new Date();
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
+
+  fs.writeFile(`src/app-${hours}_${minutes}_${seconds}.log`,
+    `${date}`,
+    (error) => {
+      if (error) {
+        console.log(error);
```

Comment:
Let's throw an error instead. Your app shouldn't continue when there is an error